compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -1101,6 +1101,7 @@ public BLangNode transform(TupleTypeDescriptorNode tupleTypeDescriptorNode) {
                 BLangSimpleVariable member = createSimpleVar(Optional.empty(), memberNode.typeDescriptor(),
                         memberNode.annotations());
                 member.setName(this.createIdentifier(null, String.valueOf(i)));
+                member.name.pos = member.typeNode.getPosition();
```

Comment:
Can we have this is above line while creating identifier instead of `null`?

Reply:
Didn't get that. Please explain

Reply:
got it. Thanks!